generate/test_data/nonest/complex.go

```diff
@@ -0,0 +1,50 @@
+package nonestTest
+
+// Code generated by github.com/GannettDigital/jstransform; DO NOT EDIT.
+
+import "time"
+
+type Complex struct {
+	Simple
+
+	Caption string `json:"caption"`
+	Credit  string `json:"credit"`
+	// The available cropped images
+	Crops          []ComplexCrops      `json:"crops"`
```

Comment:
I wonder if we should trim a trailing `s` from the split up struct type name. If you're instantiating an instance of the struct in code (like sports and elections do) then it feels strange to have the trailing `s`. On the other hand, trying to manipulate strings that represent field names can lead us down a bad path, like what I ran into with search GraphQL schemas recently. I'd imagine if we look for a lowercase `s` it would work mostly alright - if we had a field named `SSTS` then it wouldn't remove the last `s`.